ChronoVisor/include/ChronoVisorServer2.h

```diff
@@ -41,6 +45,14 @@ namespace ChronoVisor {
         std::vector<margo_instance_id> midVec_;
 
         /**
+<<<<<<< HEAD
+=======
+         * @name Clock-related variables
+         */
+        ClocksourceManager<ClockSourceCPPStyle> *CSManager;
+
+        /**
+>>>>>>> 0e4a787 (timestamps)
          * @name ClientRegistry related variables
          */
```

Comment:
merging errors

Reply:
They are not merging errors. The repository is based on the latest develop branch.